src/PointwiseFunctions/Hydro/EquationsOfState/DarkEnergyFluid.hpp

```diff
@@ -53,6 +53,13 @@ class DarkEnergyFluid : public EquationOfState<IsRelativistic, 2> {
                 "Dark energy fluid equation of state only makes sense in a "
                 "relativistic setting.");
 
+  std::unique_ptr<EquationOfState<IsRelativistic, 2>> get_clone()
+      const override {
+    auto clone = std::make_unique<DarkEnergyFluid<IsRelativistic>>(*this);
+    return std::unique_ptr<EquationOfState<IsRelativistic, 2>>(
```

Comment:
If you wanted to you can inline the `std::make_unique...` into the `std::unique_ptr<...>(std::make_unique(...))`. Up to you, just keep the different implementations consistent :)